Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -555,10 +555,9 @@ CSS::RequiredInvalidationAfterStyleChange Element::recompute_style()
         set_computed_css_values(move(new_computed_css_values));
 
     // Any document change that can cause this element's style to change, could also affect its pseudo-elements.
-    for (auto i = 0; i < to_underlying(CSS::Selector::PseudoElement::Type::KnownPseudoElementCount); i++) {
+    auto recompute_pseudo_element_style = [&](CSS::Selector::PseudoElement::Type pseudo_element) {
```

Comment:
I have a question about this: Do we need to compute and store the style _at all_ for those non-generated nodes? When I introduced this I was considering using a different enum to represent the pseudo-elements we actually need to store on the Element. Otherwise it's a fair number of bytes that Elements are carrying around.